app/src/main/java/siriuscyberneticscorporation/teachingaid47plus/SchoolClass.java

```diff
@@ -0,0 +1,78 @@
+package siriuscyberneticscorporation.teachingaid47plus;
+
+import java.util.List;
+
+/**
+ * Created by Johannes on 29.04.2016.
+ */
+public class SchoolClass {
+    private String name;
+    private String classTeacher;
+    private List<Student> students;
+    private String note;
+
+    /* Fehlt noch Fach*/
+
+    public SchoolClass(String name, String classTeacher, List<Student> students, String note) {
+        this.name = name;
+        this.classTeacher = classTeacher;
+        this.students = students;
```

Comment:
You can't initialize a list like this.
You have to do:
`this.students = new ArrayList<Student>();`